### DISCUSSION TITLE
Slingbox M2 changes port (UPnP)

### DISCUSSION THREAD

**Discussion by GR0DZ1LLA** [GerryDazoo/Slinger#355] [Q&A]
If this has been covered, I apologize, please let me know where to look.  Otherwise...

Is there a way to handle/assign/manage the port the Slingbox listens on?  The other day it was 5321, yesterday it was 5319, and over the past 24 hours, has been 5318, 5317, and now it's on 5316.  If this changes all the time, how can I properly port forward?  If it wasn't for UPnP automatically generating in the router, I wouldn't even know what port it's working on.

**Comment by sp222222** [comment]
do you have any control on the setttings on the router? You should for sure disable UPnP on the router. It's asking for trouble down the road. Yeah, it's easy to use, etc but to me not worth the security risk.  You are not using that port anyways to forward rule. you should be using the slinGER port that slingServer runs on. that port that's jumping around sounds like the slingbox port moving for some reason. that port should be found each time you launch the slinGER and logs output shows if it's not correct, what it actually is.

**Comment by GR0DZ1LLA** [comment]
Yes I have control of the router.  I really don't understand most responses here.  Anyway.  I can disable UPnP but it's my elderly parents network, and if my Dad were to buy something and it relied on UPnP and I disabled it, I'd be disappointed.  Regardless, let's say I disable UPnP.  What port am I using then?  That was the only way I got it to work, was to forward from Slinger to that specific port (5321).  Of which, I only discovered BECAUSE of UPnP.  How could Slinger possibly find that Slingbox on a remote network without that port forward?  I've been at this for weeks and still don't understand.  Also, how do I run logs for the service?  I scoured the Code readme and every release notes, and I don't see it.

**Reply by sp222222** [reply]
on your own agentdickgill - good luck
